app/common/util/src/main/java/io/syndesis/common/util/Resources.java

```diff
@@ -57,4 +60,16 @@ public static String getResourceAsText(String file, ClassLoader cl) throws IOExc
         }
     }
 
+    public static <T> Set<T> serviceLocate(Class<T> type, ClassLoader classLoader){
```

Comment:
would call it 'locateServices' or 'loadServices' (i.e. use the verb first and using plural). 'loading' seems to more appropriate as it aligns with the ServiceLoader method.

Reply:
Good point!